oci-unit-tests/helper/test_helper.sh

```diff
@@ -79,19 +140,19 @@ check_manifest_exists()
     debug -n "Listing the manifest file(s) inside ${id}"
     local files
     if ! files=$(docker exec "${id}" ls "${manifest_dir}" 2> /dev/null); then
-	debug "not found"
-	return 1
+        debug "not found"
+        return 1
     fi
     debug "done"
 
     debug "Verifying whether the manifest file(s) exist"
     for want_file in ${possible_manifest_files}; do
-	for got_file in ${files}; do
```

Comment:
I have the impression that you had removed these indentation changes during the last iteration, but now I see them back here.  Or maybe I'm misremembering?

Reply:
Yeah I included a new whitespace fixup commit that converts the tabs to spaces per our discussion to make the file's whitespace consistent.

That patch can be dropped without any conflict if you prefer keeping the tabs.

Reply:
No, it's alright.  I just thought that it'd be better to open a new PR to reindent everything, but I have no problem doing it here.  Thanks.